presto-main/src/main/java/com/facebook/presto/tdigest/TDigest.java

```diff
@@ -566,6 +566,54 @@ else if (activeCentroids == 1) {
         return weightedAverage(mean[n - 1], z1, max, z2);
     }
 
+    public double trimmedMean(double l, double h)
```

Comment:
The comments on this seem a bit sparse compared to what's included elsewhere.  Can you add a few more comments to roughly cover the different cases here?